### PR TITLE
Fixes to URI parsing with multiple parameters

### PR DESCRIPTION
**Pull request by KamuelaFranco**

### CODE REVIEW

**Comment by OlegGordiichuk** [comment]
@KamuelaFranco Hi could you pls fix Unit tests that are failing in this branch?

**Comment by KamuelaFranco** [comment]
There’s absolutely nothing failing locally, so I’m not sure.

File: Trust.xcodeproj/xcshareddata/xcschemes/Trust.xcscheme [hunk 1]
```diff
@@ -38,6 +38,11 @@
                BlueprintName = "TrustTests"
                ReferencedContainer = "container:Trust.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "InCoordinatorTests/testShowTabBar()">
```
Comment (michaelScoff) [inline Trust.xcodeproj/xcshareddata/xcschemes/Trust.xcscheme]: @KamuelaFranco can you disable this test?

Comment (KamuelaFranco) [reply Trust.xcodeproj/xcshareddata/xcschemes/Trust.xcscheme]: Done

**Pull request merged**